src/NuGet.Core/NuGet.Packaging/NuspecReader.cs

```diff
@@ -614,7 +614,8 @@ public string GetReadme()
                     var message = string.Format(
                             CultureInfo.CurrentCulture,
                             Strings.InvalidNuspecEntry,
-                            element.ToString().Trim());
+                            element.ToString().Trim(),
```

Comment:
I'm shocked we don't have more of these.

Reply:
Can we get an analyzer to break compile on these? 😄